pyverdict/pyverdict/verdictcontext.py

```diff
@@ -0,0 +1,10 @@
+import pyverdict
+
+class VerdictContext:
+    def __init__(self, gateway, host, port, database, user, pwd):
```

Comment:
Change the constructor to get the connecting string itself, e.g., "jdbc:mysql://..." It is a user's role to include a username and a password in the string.